llvm/test/Analysis/ValueTracking/knownbits-hadd-hsub.ll

```diff
@@ -0,0 +1,254 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -S -passes=instcombine < %s | FileCheck %s
```

Comment:
Maybe rename this file to knownbits-x86-hadd-hsub.ll to make it clear these are target specific

Reply:
Done.